### PR TITLE
[RW-6610][risk=no]API - DataSetController move bigQueryService calls into DataSetServiceImpl

### PR DESCRIPTION
**Pull request by NehaBroad**
---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/DataSetController.java [hunk 1]
```diff
@@ -98,7 +93,7 @@
 
   @Autowired
   DataSetController(
-      BigQueryService bigQueryService,
+      //      BigQueryService bigQueryService,
```
Comment (freemabd) [inline api/src/main/java/org/pmiops/workbench/api/DataSetController.java:96]: Need to remove this comment

File: api/src/main/java/org/pmiops/workbench/api/DataSetController.java [hunk 2]
```diff
@@ -107,7 +102,7 @@
       @Qualifier(DatasetConfig.DATASET_PREFIX_CODE) Provider<String> prefixProvider,
       GenomicExtractionService genomicExtractionService,
       WorkspaceAuthService workspaceAuthService) {
-    this.bigQueryService = bigQueryService;
+    //    this.bigQueryService = bigQueryService;
```
Comment (freemabd) [inline api/src/main/java/org/pmiops/workbench/api/DataSetController.java:105]: Need to remove this comment

**Review by freemabd — COMMENTED** [review]
Deploying now to test locally

**Review by freemabd — APPROVED** [review]
Deployed and tested locally.

File: api/src/main/java/org/pmiops/workbench/api/DataSetController.java (hunk 2 above)
Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/api/DataSetController.java]: done

File: api/src/main/java/org/pmiops/workbench/api/DataSetController.java (hunk 1 above)
Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/api/DataSetController.java]: done

**Pull request merged**